mono/mini/mini-ops.h

```diff
@@ -1381,6 +1383,9 @@ MINI_OP(OP_ARM64_CBZX, "arm64_cbzx", NONE, IREG, NONE)
 /* Branch if sreg1 != 0 */
 MINI_OP(OP_ARM64_CBNZW, "arm64_cbnzw", NONE, IREG, NONE)
 MINI_OP(OP_ARM64_CBNZX, "arm64_cbnzx", NONE, IREG, NONE)
+
+MINI_OP(OP_CLS32, "cls32", IREG, IREG, NONE)
```

Comment:
Shouldn't this be named something else ? And moved outside of #ifdef ARM64 ?

Reply:
It's Arm specific - Count leading sign bits http://infocenter.arm.com/help/index.jsp?topic=/com.arm.doc.dui0802a/a64_general_alpha.html